src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrderControllerIT.java

```diff
@@ -0,0 +1,283 @@
+                    .andExpect(status().isNotFound())
+                    .andExpect(jsonPath("$.message", is("One of the supplied UUID's was not present in the database")));
+            assertEquals(savedEntities.size(), casePaymentOrdersJpaRepository.findAllById(savedEntitiesUuids).size());
+        }
+
+        @DisplayName("Should delete entity and all its corresponding audit revisions")
+        @Test
+        void testIdPresentMultipleTimesForCreateAndUpdate() throws Exception {
+
+            CasePaymentOrderEntity savedEntity =
+                    casePaymentOrderEntityGenerator.generateAndSaveEntities(1).get(0);
+            assertTrue(casePaymentOrdersJpaRepository.findById(savedEntity.getId()).isPresent());
+
+            savedEntity.setAction("NewAction");
+            casePaymentOrdersJpaRepository.saveAndFlush(savedEntity);
+
+            mockMvc.perform(delete(CASE_PAYMENT_ORDERS_PATH).queryParam(IDS, savedEntity.getId().toString()))
+                    .andExpect(status().isOk());
+
+            assertFalse(casePaymentOrdersJpaRepository.findById(savedEntity.getId()).isPresent());
+            assertFalse(casePaymentOrdersAuditJpaRepository.findById(savedEntity.getId()).isPresent());
+        }
+
+        @DisplayName("Should fail with 400 Bad Request when invalid id (not a UUID) specified")
+        @Test
+        void shouldThrow400BadRequestWhenInvalidUuidIsSpecified() throws Exception {
+            mockMvc.perform(delete(CASE_PAYMENT_ORDERS_PATH).queryParam(IDS, "123"))
+                    .andExpect(status().isBadRequest())
+                    .andExpect(jsonPath("$.message",
+                            containsString("Failed to convert value of type 'java.lang.String'"
+                                    + " to required type 'java.util.List'")))
+                    .andExpect(jsonPath("$.details.message", is("Invalid UUID string: 123")));
+        }
+
+        @DisplayName("Should fail with 400 Bad Request when empty list of UUID's are specified")
+        @Test
+        void shouldThrow400BadRequestWhenEmptyListOfUuidsIsSpecified() throws Exception {
+            mockMvc.perform(delete(CASE_PAYMENT_ORDERS_PATH).queryParam(IDS, ""))
+                    .andExpect(status().isBadRequest())
+                    .andExpect(jsonPath("$.message", containsString("IDs are required")));
+
+        }
+
+        @DisplayName("Should fail with 400 Bad Request when CaseIds and UUIDs are specified")
+        @Test
+        void shouldThrow400BadRequestWheUuidsAndCaseIdsAreSpecified() throws Exception {
+            mockMvc.perform(delete(CASE_PAYMENT_ORDERS_PATH)
+                    .queryParam(IDS, UUID.randomUUID().toString())
+                    .queryParam(CASE_IDS, uidService.generateUID()))
+                    .andExpect(status().isBadRequest())
+                    .andExpect(jsonPath("$.message", is("Can't delete case payment order with "
```

Comment:
Use expected error constant if available.

Reply:
Resolving as code no longer applicable after recent refactored of `DELETE` to mirror `GET` endpoint validation functionality